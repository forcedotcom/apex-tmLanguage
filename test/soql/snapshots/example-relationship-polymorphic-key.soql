SELECT Id, Owner.Name FROM Task WHERE Owner.FirstName like 'B%';

SELECT Id, Who.FirstName, Who.LastName FROM Task WHERE Owner.FirstName LIKE 'B%';

SELECT Id, What.Name FROM Event;
