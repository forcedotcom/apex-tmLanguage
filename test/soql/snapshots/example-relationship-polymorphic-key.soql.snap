>SELECT Id, Owner.Name FROM Task WHERE Owner.FirstName like 'B%';
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^ source.soql keyword.query.field.apex
#         ^ source.soql punctuation.separator.comma.apex
#          ^ source.soql
#           ^^^^^^^^^^ source.soql keyword.query.field.apex
#                     ^ source.soql
#                      ^^^^ source.soql keyword.operator.query.from.apex
#                          ^ source.soql
#                           ^^^^ source.soql storage.type.apex
#                               ^ source.soql
#                                ^^^^^ source.soql keyword.operator.query.where.apex
#                                     ^ source.soql
#                                      ^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                     ^ source.soql
#                                                      ^^^^ source.soql keyword.query.field.apex
#                                                          ^ source.soql
#                                                           ^ source.soql string.quoted.single.apex punctuation.definition.string.begin.apex
#                                                            ^^ source.soql string.quoted.single.apex
#                                                              ^ source.soql string.quoted.single.apex punctuation.definition.string.end.apex
#                                                               ^^ source.soql
>
>SELECT Id, Who.FirstName, Who.LastName FROM Task WHERE Owner.FirstName LIKE 'B%';
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^ source.soql keyword.query.field.apex
#         ^ source.soql punctuation.separator.comma.apex
#          ^ source.soql
#           ^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                        ^ source.soql punctuation.separator.comma.apex
#                         ^ source.soql
#                          ^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                      ^ source.soql
#                                       ^^^^ source.soql keyword.operator.query.from.apex
#                                           ^ source.soql
#                                            ^^^^ source.soql storage.type.apex
#                                                ^ source.soql
#                                                 ^^^^^ source.soql keyword.operator.query.where.apex
#                                                      ^ source.soql
#                                                       ^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                                      ^ source.soql
#                                                                       ^^^^ source.soql keyword.operator.query.apex
#                                                                           ^ source.soql
#                                                                            ^ source.soql string.quoted.single.apex punctuation.definition.string.begin.apex
#                                                                             ^^ source.soql string.quoted.single.apex
#                                                                               ^ source.soql string.quoted.single.apex punctuation.definition.string.end.apex
#                                                                                ^^ source.soql
>
>SELECT Id, What.Name FROM Event;
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^ source.soql keyword.query.field.apex
#         ^ source.soql punctuation.separator.comma.apex
#          ^ source.soql
#           ^^^^^^^^^ source.soql keyword.query.field.apex
#                    ^ source.soql
#                     ^^^^ source.soql keyword.operator.query.from.apex
#                         ^ source.soql
#                          ^^^^^ source.soql storage.type.apex
#                               ^^ source.soql
>